src/lib/workspace/Variable.tsx

```diff
@@ -63,3 +76,19 @@ export function VariableDetails(props: Props) {
     </ErrorBoundary>
   );
 }
+
+function findMultifilterVariableLeaves(
+  variable: MultiFilterVariable,
+  variablesByParentId: Record<string, VariableTreeNode[] | undefined>
+): Variable[] {
+  if (variable.parentId == null) return [];
```

Comment:
What's this check on `variable.parentId` for?

Reply:
Oh, that's not needed. It was left over from an incorrect attempt. I will remove it.